controllers/storagecluster/reconcile.go

```diff
@@ -476,6 +476,13 @@ func (r *StorageClusterReconciler) validateStorageDeviceSets(sc *ocsv1.StorageCl
 // ensureCreated ensures that a ConfigMap resource exists with its Spec in
 // the desired state.
 func (obj *ocsCephConfig) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.StorageCluster) error {
+	var configData string
```

Comment:
We might want to call this `rookConfigData` just like the previous constant which is now `defaultRookConfigData`.